Client/Frontend/Browser/Tabs/TabTableViewCell.swift

```diff
@@ -11,6 +11,7 @@ class TabTableViewCell: UITableViewCell, Themeable {
     lazy var closeButton: UIButton = {
         let button = UIButton()
         button.setImage(UIImage.templateImageNamed("tab_close"), for: [])
+        button.accessibilityIdentifier = "closeButtonForTabTabTray"
```

Comment:
is this a typo,`TabTab`? Maybe it is not, but it sounds funny - is there a better name we could use? Maybe closeTabButtonTabTray

Reply:
No it was a typo :) I'm very bad at naming. I will change this and the other you mentioned below :) Thanks!